### PR TITLE
Added Contributing.md #2

### PR DESCRIPTION
**Pull request by rishabh-bansal**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
```
Comment (ankitjain28may) [inline CONTRIBUTING.md:6]: Can you please add the Code of Conduct to this repo separately ( Don't include in this File ) in this PR

**Comment by DEVAbhinav** [comment]
closed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+### Contributors
```
Comment (ankitjain28may) [inline CONTRIBUTING.md:84]: Please remove this content, We only want to tell them a way to contribute to this project.

**Comment by ankitjain28may** [comment]
@rishabh-bansal Open another PR with the requested changes, Thanks, Keep Coding.

**Comment by rishabh-bansal** [comment]
@ankitjain28may  I just got to know for hacktoberfest, we need to make PR during October, if possible can you please merge it during October?

**Pull request closed without merging**